meltano.yml

```
@@ -2,241 +2,287 @@ version: 1
+      kind: string
+      label: MySQL Database
+    - name: settings.mysql.password
+      env: MYSQL_PASSWORD
+      kind: password
+      label: MySQL Password
+    - name: settings.mysql.port
+      env: MYSQL_PORT
+      label: MySQL Port
+    - name: settings.mysql.ssl
+      env: MYSQL_SSL
+      kind: options
+      label: MySQL SSL
+      options:
+      - value: 'true'
+      - value: 'false'
+      - value: Amazon RDS
+    - name: settings.mysql.socket_path
+      env: MYSQL_SOCKETPATH
+      kind: string
+      label: MySQL User
+      description: |
+        This is an optional field. When using Google Cloud MySQL this is commonly required.
+       # postgres / redshift
+    - name: settings.postgres.host
+      env: POSTGRES_HOST
+      kind: string
+      label: Postgres Host
+    - name: settings.postgres.database
+      env: POSTGRES_DATABASE
+      kind: string
+      label: Postgres Database
+    - name: settings.postgres.port
+      env: POSTGRES_PORT
+      kind: string
+      label: Postgres Port
+    - name: settings.postgres.user
+      env: POSTGRES_USER
+      kind: string
+      label: Postgres User
+    - name: settings.postgres.password
+      env: POSTGRES_PASSWORD
+      kind: password
+      label: Postgres Password
+    - name: settings.postgres.ssl
+      env: POSTGRES_SSL
+      kind: options
+      description: |
+        Postgres SSL. When using Heroku this is commonly required.
+      options:
+      - value: 'true'
+      - value: 'false'
+      - value: no-verify
+    - name: settings.postgres.connection_string
+      env: POSTGRES_CONNECTIONSTRING
+      kind: password
+      label: Postgres Connection String
+      description: |
+        If your database host provides a connection string, you can use that in place of the fields above.
+       # snowflake
+    - name: settings.snowflake.account
+      env: SNOWFLAKE_ACCOUNT
+      kind: string
+      label: Snowflake Account
+    - name: settings.snowflake.username
+      env: SNOWFLAKE_USERNAME
+      kind: string
+      label: Snowflake Username
+    - name: settings.snowflake.password
+      env: SNOWFLAKE_PASSWORD
+      kind: password
+      label: Snowflake Password
+    - name: settings.snowflake.database
+      env: SNOWFLAKE_DATABASE
+      kind: string
+      label: Snowflake Database
+    - name: settings.snowflake.warehouse
+      env: SNOWFLAKE_WAREHOUSE
+        # mssql
+      kind: string
+      label: Snowflake Warehouse
+    - name: settings.mssql.host
+      env: MSSQL_HOST
+      kind: string
+      label: MSSQL Host
+    - name: settings.mssql.database
+      env: MSSQL_DATABASE
+      kind: string
+      label: MSSQL database
+    - name: settings.mssql.user
+      env: MSSQL_USER
+      kind: string
+      label: MSSQL user name
+    - name: settings.mssql.password
+      env: MSSQL_PASSWORD
+      kind: password
+      label: MSSQL Password
+    - name: settings.mssql.trust_server_certificate
+      env: MSSQL_CERTIFICATE
```

Evidence seems to expect these settings to be nested inside an `options` object:

```suggestion
    - name: settings.mssql.options.trust_server_certificate
      env: MSSQL_TRUST_SERVER_CERTIFICATE
```

https://github.com/evidence-dev/evidence/blob/main/packages/mssql/index.cjs#L101-L105